.gitignore

```diff
@@ -1,4 +1,3 @@
 # .gi/tignore
 /node_modules
```

Comment:
- [ ] (OPTIONAL) Let's remove this commented code in order to make our work clean 👍